lib/use_case/question/create.rb

```diff
@@ -20,7 +20,8 @@ def call
           content:             content,
           author_is_anonymous: anonymous,
           author_identifier:   author_identifier,
-          user:                source_user_id.nil? ? nil : source_user
+          user:                source_user_id.nil? ? nil : source_user,
+          direct:              true
```

Comment:
would be cool to have this explicitly in `CreateFollowers` as well

Reply:
so, that it explicitly sets `direct: false`? Because follower questions are never direct.

Reply:
exactly